src/test/java/calculator/CalculatorFactoryTest.java

```diff
@@ -0,0 +1,74 @@
+package calculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class CalculatorFactoryTest {
+
+
+    @Test
+    void 덧셈() {
+        CalculatorFactory calculator = new CalculatorFactory();
+
+        assertThat(
+                calculator.handleWayByOperator(
+                        "+",
+                        1,
+                        2
+                )
+        ).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(calculator.handleWayByOperator("+", 1, 2)).isEqualTo(3);
```
위와 같이 구현하니 라인만 차지하고 가독성이 떨어지는 느낌이 드는데 위와 같이 구현하는 것은 어떨까?